app.js

```diff
@@ -22,6 +23,11 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 indexRouter(app);
 
+// sync Sequelize with MySQL Database
+(async () => {
+  await db.sequelize.sync();
+})();
+
```

Comment:
It's better to use Sequelize migrations to update the database schemas. Sync may not work sometimes when you change a model schema, it just creates the tables that don't exist in the database